cluster/manager/manager.go

```diff
@@ -1482,8 +1482,12 @@ func (c *ClusterManager) StartWithConfiguration(
 		QuorumTimeout:    quorumTimeout,
 		SuspicionMult:    types.DEFAULT_SUSPICION_MULTIPLIER,
 	}
+
+	nodeIp := net.JoinHostPort(c.selfNode.DataIp, c.gossipPort)
+	logrus.Infoln("Init gossip: ", nodeIp)
+
 	c.gossip = gossip.New(
-		c.selfNode.DataIp+":"+c.gossipPort,
+		nodeIp,
```

Comment:
nit -- why are they called "node IPs"?   (they are "host+port" combo's -- they should be called "endpoints", or maybe "hostports"?)